index.js

```diff
@@ -2,10 +2,14 @@
 const supportsColor = require('supports-color');
 const hasFlag = require('has-flag');
 
+/**
+@param {string} versionString
+@returns {{ major: number, minor: number, patch: number }}
+*/
 function parseVersion(versionString) {
 	if (/^\d{3,4}$/.test(versionString)) {
 		// Env var doesn't always use dots. example: 4601 => 46.1.0
-		const m = /(\d{1,2})(\d{2})/.exec(versionString);
+		const m = /(\d{1,2})(\d{2})/.exec(versionString) ?? [];
```

Comment:
You need to either upgrade XO or not use `??`. I suggest not using `??` as that's a less invasive change.